src/routing/comm.rs

```diff
@@ -364,9 +376,16 @@ mod tests {
         let invalid_addr = get_invalid_addr().await?;
 
         let message = Bytes::from_static(b"hello world");
-        let (result, failed_recipients) = comm.send(&[invalid_addr], 1, message.clone()).await;
-        assert!(result.is_err());
-        assert_eq!(failed_recipients, [invalid_addr]);
+        let status = comm.send(&[invalid_addr], 1, message.clone()).await?;
+
+        assert_matches!(
+            &status,
+            &SendStatus::MinDeliveryGroupSizeFailed(_)
```

Comment:
can you do `SendStatus::MinDeliveryGroupSizeFailed([invalid_addr])` , and avoid the below if clause?